server/routes/student/index.ts

```diff
@@ -1,5 +1,9 @@
 import { Router } from 'express';
+import { checkUserAttendQuiz, addPrivateQuizScore, sendEmail } from '../../controllers';
+import { checkAuth } from '../../middlewares';
 
 const router = Router();
 
+router.post('/score', checkAuth('student'), checkUserAttendQuiz, addPrivateQuizScore, sendEmail);
```

Comment:
checUserAttendQuiz controller, should be in route `router.get('/quiz-data/:quizId', checUserAttendQuiz, quizDetails')`
so the user he can know before start the exam that he submitted it